### DISCUSSION TITLE
Gopro 11 sensor gone dark after  making LOGB 200,-128 permenant and restarting

### DISCUSSION THREAD

**Discussion by carbendi** [gopro/labs#220] [Q&A] [answered]
Hi 
GoPro 11 sensor gone near dark like it is locked at a super high shutter speed after  making LOGB 200,-128 permenant and restarting. Did I brake something or do you guys think it is fixable? 
It would be nice to have a way to apply qr codes through text files on sd card :) 

Factory reset, updating firmware didn't work. 

And here is a sample video file recorded by the problematic GoPro11 that is showing the problem.
https://www.dropbox.com/sh/6e7z8iddumpsfnu/AAAB1UXTSj2OERy6U4AZ-Es-a?dl=0

**Comment by dnewman-gpsw** [comment, marked as answer]
The camera is recoverable.  Add an empty **qr_reset.txt** file into the MISC on the SD Card (or just rename the version.txt file.) Add it to the camera.  With the camera off, provide USB power.  The camera should auto boot, read the qr_reset.txt file, and remove all !M saved settings.  I'm now looking into which command failed, causing the issue.

**Reply by dnewman-gpsw** [reply]
You will need to boot twice, the first time clears the settings, the second time will be with no Labs hacks applied.

**Reply by carbendi** [reply]
Thanks so much for this. I was really afraid I broke something in my naked gopro 11 :D
This fixed the issue.

**Comment by andrerav** [comment]
Interesting. I might add that I've also had my GoPro go dark like this on rare occasions only when using LOGB=X,Y, but never with just LOGB=X. Also my GoPro fixes itself after a restart.

**Reply by dnewman-gpsw** [reply]
I have found the issue. It is a race condition, with the LOG curve before the GP2 is ready.  This is way only !MLOGB can do it, but even this is not every time or on every camera.  Next release will delay LOGB configuration until the hardware is ready.

**Reply by andrerav** [reply]
Fantastic, nice job!
